src/client/ClientDataResolver.js

```diff
@@ -244,7 +244,15 @@ class ClientDataResolver {
         if (/^https?:\/\//.test(resource)) {
           request.get(resource)
             .set('Content-Type', 'blob')
-            .end((err, res) => err ? reject(err) : resolve(res.body));
+            .end((err, res) => {
```

Comment:
Isn't it more proper to do
`if (err) return reject(err);`
though, for consistency with cases where you'd need to do that to prevent execution of additional code?